src/actions/userActions.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import { SET_SIGNUP_FORM_INPUT, SET_SUBMIT_USERS, USER_ERRORS } from './type';
+
+export const setSignupInput = data => async (dispatch) => {
+  const url = '/api/users';
+  return axios.post(url, data).then((res) => {
+    dispatch({
+      type: SET_SIGNUP_FORM_INPUT,
+      payload: res.data
+    });
```

Comment:
create a function for the
``` 
{
type: SET_SIGNUP_FORM_INPUT,
 payload: res.data
    }  
```  and call it in the dispatch improve readability.

Reply:
true